pkg/server/proto/server.proto

```diff
@@ -1604,6 +1604,26 @@ message Job {
   // on an ODR spawned from this config.
   Ref.OnDemandRunnerConfig ondemand_runner = 112;
 
+  // This is optional and overrides the task that is used by the ondemand
```

Comment:
I didn't fix up the doc comments in the test package, but since I think these end up being user and dev facing, we should at least align those to be consistent.